### PR TITLE
Changing struct

### PR DESCRIPTION
**Pull request by NicoKos**

### CODE REVIEW

File: ietf-document/draft-0rtt-bdp/draft-kuhn-quic-0rtt-bdp-09.xml [hunk 1]
```diff
@@ -211,12 +108,53 @@
...
+		</section>
+
+		<section anchor="subsec:scope" title="Scope and structure of the document">
+
+			<t> The following scenario is considered through the whole document:<list style="numbers">
+				<t>Sharing 1-RTT transport parameters for 0-RTT connections : <xref target="sec:implem"></xref> discusses ways of implementing this in QUIC. Interoperability is required for this transport parameter exchange.</t>
+				<t>Safely using previously measured transport parameters in 0-RTT connections: <xref target="subsec:rationale_jump"></xref> discusses solutions that can enable a safe increase of the transmission rate. These safety nets SHOULD be implemented when the transport parameters are used to increase the ingress initial transmission rate.</t>
+			</list></t>
+
+			<t>The following example could be proposed by using the different approaches described in <xref target="sec:implem"></xref> and <xref target="subsec:rationale_jump"></xref>:<list style="symbols">
```
[the remark this answers is not in the document]
Comment (gorryfair) [reply ietf-document/draft-0rtt-bdp/draft-kuhn-quic-0rtt-bdp-09.xml:124]: I committed small changes in English/RFC-style directly to the branch.

[the remark this answers is not in the document]
Comment (NicoKos) [reply ietf-document/draft-0rtt-bdp/draft-kuhn-quic-0rtt-bdp-09.xml:124]: The idea is that the proposals of section 3 and section 4 can be mixed independently. 
An example on how they can be mixed is proposed here - 

"The following design can result in the exploitation of the solution proposed in section 3 and section 4"

[the remark this answers is not in the document]
Comment (gorryfair) [reply ietf-document/draft-0rtt-bdp/draft-kuhn-quic-0rtt-bdp-09.xml:124]: Ah OK, maybe just needs more explanation.

[the remark this answers is not in the document]
Comment (NicoKos) [reply ietf-document/draft-0rtt-bdp/draft-kuhn-quic-0rtt-bdp-09.xml:124]: Would the following wording be more clear 

"The following design can result in the exploitation of the solution proposed in section 3 and section 4" ?

[the remark this answers is not in the document]
Comment (NicoKos) [reply ietf-document/draft-0rtt-bdp/draft-kuhn-quic-0rtt-bdp-09.xml]: This is proposed in commit https://github.com/NicoKos/QUIC_HIGH_BDP/pull/123/commits/1091d66a8853a23d8d5fbc15768f1171ca735167

**Pull request merged**